sys/linux/dev_vfio.txt

```diff
@@ -0,0 +1,51 @@
+# Copyright 2017 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+# https://www.kernel.org/doc/Documentation/vfio.txt
+# https://elixir.bootlin.com/linux/latest/source/drivers/vfio/vfio.c
+# https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/vfio.h
+# https://elixir.bootlin.com/linux/latest/source/drivers/vfio/vfio_iommu_type1.c
+
+include <uapi/linux/fcntl.h>
+include <linux/vfio.h>
```

Comment:
Use `uapi/linux/vfio.h` path. Sometimes the same header is present both as include/linux/foo.h and include/uapi/linux/foo.h. So it's better to be explicit.